pkgs/test/nixpkgs-check-by-name/README.md

```diff
@@ -0,0 +1,96 @@
+# Nixpkgs pkgs/by-name checker
+
+This directory implements a program to check the [validity](#validity-checks) of the `pkgs/by-name` Nixpkgs directory.
+This is part of the implementation of [RFC 140](https://github.com/NixOS/rfcs/pull/140).
+
+## API
+
+This API may be changed over time if the CI making use of it is adjusted to deal with the change appropriately, see [Hydra builds](#hydra-builds).
+
+- Arguments:
+  - `<NIXPKGS>`: The path to the Nixpkgs to check
```

Comment:
`NIXPKGS` hasn't been defined yet.

I suppose it's just a positional argument.


```suggestion
- Command line: `nixpkgs-check-by-name <NIXPKGS>`
- Arguments:
  - `<NIXPKGS>`: The path to the Nixpkgs to check
```